.travis.yml

```diff
@@ -0,0 +1,13 @@
+dist: xenial
+sudo: true
+language: python
+script:
+    - set -e
+    - export TAG=`if [ "$TRAVIS_PULL_REQUEST_BRANCH" == "" ]; then echo "latest"; else echo branch-$(echo $TRAVIS_PULL_REQUEST_BRANCH | sed s:/:-:g); fi`
+    - docker build -t eu.gcr.io/census-eq-ci/eq-survey-runner-benchmark:"$TAG" .
+    - echo "$DOCKER_GCR_PASSWORD" | base64 --decode | docker login -u "$DOCKER_GCR_USERNAME" --password-stdin https://eu.gcr.io
```

Comment:
This docker push will fail until we have appropriate credentials setup in travis for this repo

Reply:
Ill update this shortly.

Reply:
Done.